tests/console/redis.pm

```diff
@@ -131,6 +132,15 @@ sub test_redis {
 sub run {
     my $self = shift;
     select_serial_terminal;
+
+    # For redis is removed from 15-SP7, so the log directory doesn't exist, which will cause failure.
+    # So create this directory as a workaround.
+    my $log_dir = "/var/log/redis/";
+    if (script_run("test -d $log_dir") != 0) {
+        record_info("Create the redis log directory as a workaround");
```

Comment:
IMO it's better to use short word e.g. workaround or poo#179867 with whatever longer text as second field.
Longer text is not readable, only visible as pop-up when keeping mouse over the window.